packages/browser-repl/src/components/editor.tsx

```diff
@@ -93,7 +92,8 @@ export class Editor extends Component<EditorProps> {
         enableLiveAutocompletion: !!this.props.autocompleter,
         enableSnippets: false,
         showLineNumbers: false,
-        tabSize: 2
+        tabSize: 2,
+        useWorker: false
```

Comment:
Compass is already using this. This side-steps the webpack build problem now. Workers are separate entry points so are separate files that you have to build and serve up. That requires extra config to get working.

How did this work before? Ace seems to fall back if the webworker errors. In storybook and in tests everything would work and pass it would just also log the 404 as it tries to load the worker and it fails.

Reply:
https://jira.mongodb.org/browse/COMPASS-5328